HopsanGUI/GUIObjects/GUIContainerObject.cpp

```diff
@@ -115,7 +115,7 @@ void SystemObject::commonConstructorCode()
     mIsCreatingConnector = false;
     mShowSubComponentPorts = gpMainWindow->mpTogglePortsAction->isChecked();
     mShowSubComponentNames = gpMainWindow->mpToggleNamesAction->isChecked();
-    mSignalsHidden = gpMainWindow->mpToggleSignalsAction->isChecked();
+    mSignalsHidden = !gpMainWindow->mpToggleSignalsAction->isChecked();
```

Comment:
Maybe it would have been better to rename the variable to mSignalsVisible (instead of having bool with negative meaning), but I guess this works to. Odd that we have not seen this bug before. 
No need to change unless you want to, then all places where this is used must be changed to.

Reply:
Agreed. This has been confusing for a long time. If the action is toggled, the boolean should become true.